x-pack/test/reporting_api_integration/reporting_and_security/spaces.ts

```diff
@@ -39,19 +39,24 @@ export default function ({ getService }: FtrProviderContext) {
     );
   };
 
-  const spacesSharedObjectsArchive =
-    'x-pack/test/functional/es_archives/reporting/ecommerce_kibana_spaces';
-
   describe('Exports and Spaces', () => {
+    const ids = ['non_default_space', 'non_timezone_space'];
```

Comment:
It doesn't look like `non_timezone_space` is ever used in the tests?  Might be good to check with the test author.  It could be that they intended to create tests for non_timezone_space and never did?  Otherwise this could be simplified.

Reply:
Hey boss @LeeDr , I've reached out to @tsullivan